addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties

```diff
@@ -0,0 +1,9 @@
+paramminer.api.action.helloWorld = Logs "hello world called" with debug level.
```

Comment:
All the other Messages<locale>.properties files can be removed. They'll be taken care of by crowdin.